pkg/admin/util.go

```diff
@@ -72,13 +82,22 @@ func New(endpoint, accessKey, secretKey, pcEndpoint, pcUsername, pcPassword, acc
 		accountName = "ntnx-cosi-iam-user"
 	}
 
-	// If no client is passed initialize it
-	if httpClient == nil {
-		// SSL certificate verification turned off
-		tr := &http.Transport{
-			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-		}
-		httpClient = &http.Client{Transport: tr}
+	tlsConfig := TlsConfig{
+		CACert:   caCert,
+		Insecure: false,
+	}
+
+	if insecure == "true" {
+		tlsConfig.Insecure = true
```

Comment:
Make `insecure` a bool
```suggestion
		Insecure: insecure,
	}
```

Reply:
Lets do this. Use bool here.